backend/routes/api.php

```diff
@@ -73,6 +73,12 @@
             Route::get('/', 'VisitController@getPageViews');
         });
 
+        Route::group([
```

Comment:
why did you add group? There will not be any of additional endpoints for chart values of total visitors